client/src/components/AuthPage/Background.jsx

```diff
@@ -0,0 +1,492 @@
+        <span
+          style={{
+            boxShadow:
+              "0 0 0 10px #ff2d7544, 0 0 50px #ff2d75,0 0 100px #ff2d75",
+            animationDuration: "calc(150s/21)",
+          }}
+          className="p-[5px] animate-linear_bubble flex items-center justify-center shadow-lg relative w-[24px] h-[24px] bg-bubble1 mr-[13px] ml-[13px] rounded-full"
+        >
+          <IoMdMusicalNote color="white" />
+        </span>
+        <span
+          style={{
+            boxShadow:
+              "0 0 0 10px #4fc3dc44, 0 0 50px #4fc3dc,0 0 100px #4fc3dc",
+            animationDuration: "calc(150s/26)",
+          }}
+          className="p-[5px] animate-linear_bubble flex items-center justify-center shadow-lg relative w-[24px] h-[24px] bg-bubble mr-[13px] ml-[13px] rounded-full"
+        >
+          <BsMusicNoteBeamed color="white" />
+        </span>
+        <span
+          style={{
+            boxShadow:
+              "0 0 0 10px #ff2d7544, 0 0 50px #ff2d75,0 0 100px #ff2d75",
+            animationDuration: "calc(150s/25)",
+          }}
+          className="p-[5px] animate-linear_bubble flex items-center justify-center shadow-lg relative w-[24px] h-[24px] bg-bubble1 mr-[13px] ml-[13px] rounded-full"
+        >
+          <IoMdMusicalNote color="white" />
+        </span>
+        <span
+          style={{
+            boxShadow:
+              "0 0 0 10px #4fc3dc44, 0 0 50px #4fc3dc,0 0 100px #4fc3dc",
+            animationDuration: "calc(150s/23)",
+          }}
+          className="p-[5px] animate-linear_bubble flex items-center justify-center shadow-lg relative w-[24px] h-[24px] bg-bubble mr-[13px] ml-[13px] rounded-full"
+        >
+          <BsMusicNoteBeamed color="white" />
+        </span>
+        <span
+          style={{
+            boxShadow:
+              "0 0 0 10px #ff2d7544, 0 0 50px #ff2d75,0 0 100px #ff2d75",
+            animationDuration: "calc(150s/22)",
+          }}
+          className="p-[5px] animate-linear_bubble flex items-center justify-center shadow-lg relative w-[24px] h-[24px] bg-bubble1 mr-[13px] ml-[13px] rounded-full"
+        >
+          <IoMdMusicalNote color="white" />
+        </span>
+        <span
+          style={{
+            boxShadow:
+              "0 0 0 10px #4fc3dc44, 0 0 50px #4fc3dc,0 0 100px #4fc3dc",
+            animationDuration: "calc(150s/16)",
+          }}
+          className="p-[5px] animate-linear_bubble flex items-center justify-center shadow-lg relative w-[24px] h-[24px] bg-bubble mr-[13px] ml-[13px] rounded-full"
+        >
+          <BsMusicNoteBeamed color="white" />
+        </span>
+        <span
+          style={{
+            boxShadow:
+              "0 0 0 10px #ff2d7544, 0 0 50px #ff2d75,0 0 100px #ff2d75",
+            animationDuration: "calc(150s/14)",
+          }}
+          className="p-[5px] animate-linear_bubble flex items-center justify-center shadow-lg relative w-[24px] h-[24px] bg-bubble1 mr-[13px] ml-[13px] rounded-full"
```

Comment:
mr-[13px] ml-[13px] can be mx-[13px] I guess. Is there any specific reason for writing like this? This is written like this in many lines.